jinahub/indexers/searcher/compound/FaissPostgresIndexer/README.md

```diff
@@ -32,11 +36,68 @@ with:
 
 - from the `Flow.rolling_update` method. See [docs](https://docs.jina.ai/advanced/experimental/indexers/).
 
-The folder needs to contain the data exported from your Indexer. Again, see [docs](https://docs.jina.ai/advanced/experimental/indexers/).
+The folder needs to contain the data exported from your Indexer.
+Again, see [docs](https://docs.jina.ai/advanced/experimental/indexers/).
+
+Check [integration tests](https://github.com/jina-ai/executors/tree/main/tests/integration/psql_dump_reload) for an example on how to use it.
+
+### Direct import from PSQL
+
+Alternatively, you can use the import from PSQL feature.
+
+
+This allows you to bypass the expensive writing to disk operation.
+It also removes the complexities that can arise out of doing this in a cloud environment, where you would need to mount the volume in one instance, and then migrate it, and then perform the import.
```

Comment:
Now that we have line breaks I see in one glance, that this sentence is too long :D

Reply:
adapted